app/api/search/route.ts

```diff
@@ -1,7 +1,13 @@
 import { NextResponse, NextRequest } from "next/server";
 import discogs from "@/app/utils/discogs";
+import { SearchResponseSchema } from "@/app/utils/discogs.schemas";
```

Comment:
`SearchResponseSchema` can go in this file since it belongs and is associated to your API, not discog's

Reply:
I created a file called collections.schemas.ts where I defined SearchAlbumSchema, the Album type, and SearchResponseSchema. Is that ok or should I still put SearchResponseSchema in the search API route.ts file?